ops/mappings.json

```diff
@@ -588,6 +714,48 @@
         "mapping": "FundingRequest",
         "arcVersion": "0.1.2-rc.0"
       },
+      {
+        "name": "DAORegistry",
+        "contractName": "DAORegistryInstance",
+        "dao": "package",
+        "mapping": "DAORegistry",
+        "arcVersion": "0.1.2-rc.2"
+      },
+      {
+        "name": "DAOFactory",
+        "contractName": "DAOFactoryInstance",
+        "dao": "package",
+        "mapping": "DAOFactory",
+        "arcVersion": "0.1.2-rc.2"
+      },
+      {
+        "name": "GenesisProtocol",
+        "contractName": "GenesisProtocol",
+        "dao": "package",
+        "mapping": "GenesisProtocol",
+        "arcVersion": "0.1.2-rc.2"
+      },
+      {
+        "name": "DAORegistry",
+        "contractName": "DAORegistryInstance",
+        "dao": "package",
+        "mapping": "DAORegistry",
+        "arcVersion": "0.1.2-rc.4"
```

Comment:
this is rc.4 ?!

Reply:
I returned both, though I'm not sure we actually have any daos using 4, should I keep it anyway or remove it?

Reply:
keep it anyhow

Reply:
ok, so please just approve :)